src/main.js

```diff
@@ -0,0 +1,40 @@
+import {createUserInfoTemplate} from './view/user-Info';
```

Comment:
все файлы должны быть в нижнем регистре

```suggestion
import {createUserInfoTemplate} from './view/user-info';
```